lib/mongo_mapper/utils.rb

```diff
@@ -8,5 +8,17 @@ def self.get_safe_options(options)
       safe = {:w => safe} if safe.is_a? Integer
       safe
     end
+
+    def self.filter_param(name, raw_value)
+      if ActiveSupport.version >= Gem::Version.new("6.0")
+        @filter ||= begin
+          require "active_support/parameter_filter"
```

Comment:
parameter_filter exists in ActionPack in Rails 5.x.
Then, we needed to check the version.

Rails 5.x was already EOL.
So we suggest dropping the support for Rails 5.x to keep the implementation simple.

> Rails 5.2.Z is included in the list of supported series until June 1st 2022.

https://guides.rubyonrails.org/maintenance_policy.html#severe-security-issues

Reply:
Great feature!  

I think we should leave in the 5.x support for now since it was so recently removed and since the conditional code is already there.  I suspect there are also a lot of people running rails 5.2.  (looks like ~ 17% are on 5.2 - https://rails-hosting.com/2022/#what-versions-of-rails-are-you-using-in-your-applications).

With that said, I think in the next few months we should drop rails 5.2 support, but only do it with a major version upgrade.

Also, I suspect that once the rails team stops providing security updates to rails 5.2 it will encourage a majority of users to upgrade.

So let's leave in the conditional code for now but plan to remove it in the next few months with mongomapper v16

Reply:
Okay, thanks for your comment based on the evidence 👍 
Let's keep the conditional code!

> it in the next few months with mongomapper v16

👍 👍 👍

Reply:
Sounds good 👍 
I will add a deprecation warning for v16.